src/main/java/com/denizenscript/clientizen/objects/EntityTag.java

```diff
@@ -63,6 +65,7 @@ public class EntityTag implements ObjectTag, Adjustable {
     public final UUID uuid;
     public Entity entity;
     public final boolean isFake;
+    private static final List<UUID> visibleEntities = new ArrayList<>();
```

Comment:
Probably makes sense for this to be a `Set`? it only contains a UUID once, order doesn't matter, and it's used for relatively rapid `contains` checking